README.md

```diff
@@ -70,6 +231,234 @@ It can be useful to put RPKs into an `rpk` sub-directory of your current Houdini
+
+Default is Houdini 19.5. See below how to build for different Houdini versions.
+
+### Bootstrap
+
+The below steps will populate your local Conan repository with dependencies for the Palladio build system. You only need to work through this section once (or if you want to upgrade one of the dependencies).
+
+#### Linux
+1. Checkout Palladio: `git clone git@github.com:esri/palladio.git && cd palladio`
+1. Download CityEngine SDK: `conan create -pr conan/profiles/linux-gcc93 conan/cesdk cesdk/2.6.8300@esri-rd-zurich/stable`
+1. Extract and package the HDK from your local Houdini 19.5 installation (adjust Z to your Houdini version): `conan create -pr conan/profiles/linux-gcc93 conan/houdini houdini/19.5.Z@sidefx/stable` (Note: use the option `-e HOUDINI_INSTALL=/path/to/your/hfs19.5.Z`, if Houdini is not installed at the standard location, e.g. at `/opt/hfs19.5.Z` for Linux).
+
+#### Windows
+1. Checkout Palladio: `git clone git@github.com:esri/palladio.git`
+1. Open a Windows command shell and `cd` to the Palladio git repository
+1. Download CityEngine SDK: `conan create -pr conan/profiles/windows-v142 conan/cesdk cesdk/2.6.8300@esri-rd-zurich/stable`
+1. Extract and package the HDK from your local Houdini installation (adjust Z to your Houdini version): `conan create -pr conan/profiles/windows-v142 conan/houdini houdini/19.5.Z@sidefx/stable` (Note: use the option `-e HOUDINI_INSTALL=C:/path/to/your/houdini/installation`, if Houdini is not installed at the standard location for Windows).
+
+### Building Palladio
+
+Note: to build for another Houdini version, add the cmake argument `-DPLD_HOUDINI_VERSION=X.Y`.
+
+#### Linux
+1. Ensure GCC 9.3 is active.
+1. `cd` into your Palladio git repository
+1. `mkdir -p build/release && cd build/release`
+1. `cmake -DCMAKE_BUILD_TYPE=Release ../../src`
+1. `make install` (the plugin will be installed into your `~/houdini19.5/dso` directory)
+
+#### Windows
+1. Open a MSVC 14.2 x64 shell (Visual Studio 2019) and `cd` to the Palladio git repository
+1. `mkdir build/release`
+1. `cd build/release`
+1. `cmake -G "NMake Makefiles" -DCMAKE_BUILD_TYPE=Release ../../src`
+1. `nmake install` (the plugin will be installed into your `%USERPROFILE%/Documents/houdini19.5/dso` directory)
+
+### Running Palladio
+See [Quick Start](#quick-start) how to launch Houdini with Palladio.
+
+### Building and Running Unit Tests
+
+#### Linux
+1. Ensure GCC 6.3 is active.
+1. `cd` into your Palladio git repository
+1. `mkdir -p build/relTest && cd build/relTest`
+1. `cmake -DCMAKE_BUILD_TYPE=Release -DPLD_TEST=1 ../../src`
+1. `make palladio_test`
+1. run `bin/palladio_test`
+
+#### Windows
+1. Open a MSVC 14.2 x64 shell (Visual Studio 2019) and `cd` to the Palladio git repository
+1. `mkdir build/relTest`
+1. `cd build/relTest`
+1. `cmake -G "NMake Makefiles" -DCMAKE_BUILD_TYPE=Release -DPLD_TEST=1 ../../src`
+1. `nmake palladio_test`
+1. ensure that the `bin` subdirectory of your Houdini installation is in the `PATH`
```

Comment:
this is a leftover from when the tests still depended on Houdini, please remove this line